.github/workflows/ci_house_keeping.yaml

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright Confidential Containers Contributors 2023.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run clean up routines periodically.
+---
+name: CI house keeping
+
+on:
+  schedule:
+    # Runs at 04:00 (UTC time) every day
+    - cron: '0 4 * * *'
```

Comment:
I can see 19 closed PRs within the last week and 56 within last month, wouldn't a weekly upkeep be more suitable?

Reply:
Good idea @ldoktor! I will change it.